layouts/partials/footer.html

```diff
@@ -18,7 +18,7 @@
         {{ end }}
       </div>
       <div class="col-12 col-sm-4 text-center py-2 order-sm-2">
-        {{ with .Site.Params.copyright }}<small class="text-white">&copy; {{ now.Year}} {{ . }}
+        {{ with .Site.Params.copyright }}<small class="text-white">&copy; 2013-present {{ . }}
```

Comment:
I'd vote to keep the dynamically generated year (just because we can in this partials file, whereas we can't in the static LICENSE file), so:

    2013-{{ now.Year }}

Reply:
I agree, I'll make that update.